### DISCUSSION TITLE
AudioMotion for Blazor.

### DISCUSSION THREAD

**Discussion by fgilde** [hvianna/audioMotion.js#45] [Show and tell]
Hi I created a blazor wrapper component for audioMotion I named it AuralizeBlazor. Hope you like it. 

[Live Demo](https://auralizeblazor.azurewebsites.net/) | [GitHub](https://github.com/fgilde/AuralizeBlazor)

![alt text](https://raw.githubusercontent.com/fgilde/AuralizeBlazor/master/AuralizeBlazor/screenshot1.png)

AuralizeBlazor provides a audio visualizer component named `Auralizer`.
This a Blazor component that integrates the powerful features of the audioMotion.js library, enabling developers to incorporate real-time audio visualization into Blazor applications with ease. This document provides an overview of how to use the component, including its properties and methods.

## Getting Started

To use `AudioMotion` in your Blazor application, first ensure the component library is added to your project. Then, you can incorporate the visualizer into your pages or components.

### 1. Installation

Make sure you have installed the `AuralizeBlazor` package in your project. If it's not installed, add it via NuGet package manager, CLI or PackageReference.

```bash
dotnet add package AuralizeBlazor
```
```xml
<PackageReference Include="AuralizeBlazor" Version="*" />
```

### 2. Usage

Add the namespace to your `_Imports.razor`:

```c#
@using AuralizeBlazor
```

### 3. Example
After adding the namespace, you can use the `Auralizer` component in your pages or components. Here's an example of how to use the component:

```html
<Auralizer        
    Radial="true"
    Height="400px"
    Width="100%">    
    <audio src="path/to/audio1.mp3" controls></audio>
</Auralizer>

```

**Comment by hvianna** [comment]
I love this! Awesome job!
